modules/imgproc/src/color.cpp

```diff
@@ -173,7 +148,7 @@ static void splineBuild(const softfloat* f, int n, float* tab)
     int i;
     tab[0] = tab[1] = 0.0f;
 
-    for(i = 1; i < n-1; i++)
+    for(i = 1; i <= n-1; i++)
```

Comment:
"<= n" ? (this doesn't look similar to line 150)

Reply:
This is a fix of an uninitialized memory access due to an error in the _splineBuild_ function. It was not visible when initialization was static, but it is detected by valgrind when initialization is dynamic.